packages/block-editor/src/components/block-list/content.scss

```diff
@@ -431,7 +431,6 @@
 }
 
 .block-editor-iframe__body {
-	background-color: $white;
```

Comment:
@ntsekouras This isn't a good solution because now this rule has a higher specificity than a theme just using the `body` selector. Also, if a theme sets it on `html`, it won't be visible. We need to set the white background on the `iframe` element so that it's not transparent. I fixed that below.

Reply:
In this brunch this issue https://github.com/WordPress/gutenberg/issues/46291 doesn't work though since we're scaling the `body` of the iframe and this is where the background is applied from theme.json. Any ideas to work around that?

Reply:
Ah the scaling... I just talked to Riad about that the other day. I think we should moved the scaling outside the iframe. I'll make a separate PR for that then first.

Reply:
Would love to see this fix merged! It's breaking a lot of themes in our deployment of Gutenberg, so we've had to remove this line of CSS from our deployment. It seems like load order of CSS (for example, if CSS is concatenated) can break this.

Reply:
Ah nice, @fullofcaffeine found that this PR fixes the issue: https://github.com/WordPress/gutenberg/pull/46732